src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+  imdbId,
+}) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          // eslint-disable-next-line max-len
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src={imdbUrl} alt="imdb" />
```

Comment:
```suggestion
            <img src="images/imdb-logo.jpeg" alt="imdb" />
```
Це просто логотипчик, вн статичний, по imdbUrl немає картинки